client/signup/config/steps-pure.js

```diff
@@ -441,7 +441,7 @@ export function generateSteps( {
 			apiRequestFunction: createSiteOrDomain,
 			props: {
 				get headerText() {
-					return i18n.translate( 'Choose your site?' );
+					return i18n.translate( 'Choose your site' );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.